src/pfe/file-watcher/server/test/scripts/exec.sh

```diff
@@ -25,24 +25,36 @@ EOF
 }
 
 function clone () {
-    curl -H "Content-type: application/json" -d '{"projectName": "'$1'", "parentPath": "'$2'", "url": "'$3'"}' "http://localhost:9090/api/v1/projects"
+    if [ $TEST_TYPE == "local" ]; then
+        curl -H "Content-type: application/json" -d '{"projectName": "'$1'", "parentPath": "'$2'", "url": "'$3'"}' "http://localhost:9090/api/v1/projects"
+    elif [ $TEST_TYPE == "kube" ]; then
+        kubectl exec -i $CODEWIND_POD_ID -- curl -H "Content-type: application/json" -d '{ "projectName": "'$1'" , "parentPath": "'$2'", "url": "'$3'" }' "https://localhost:9191/api/v1/projects" --insecure
```

Comment:
I am thinking can we make a function get the curl call? So in the future if the project creation parameters change we only need to change it in one place. And also dependening on if test type is kube or local, it will dynamically select the protocol and ports.

Reply:
no, the curl needs to run inside the kube bash terminal

Reply:
I meant something like this:
```
function getCurl() {
  protocol="http"
  port="9090"
  if [ $1 == "kube" ]; then
      protocol="https"
      port="9091"
      flags="--insecure"
  fi
  curl -H "Content-type: application/json" -d '{"projectName": "'$2'", "parentPath": "'$3'", "url": "'$4'"}' "$protocol://localhost:$port/api/v1/projects" $flags
}

Need to test the function out but hopefully you get the idea behind it?

Reply:
done